cli/src/org/partiql/cli/main.kt

```diff
@@ -139,6 +142,7 @@ private val outputFormatOpt = optParser.acceptsAll(listOf("output-format", "of")
  *      * -w --wrap-ion: wraps Ion input file values in a bag, requires the input format to be ION, requires the query option
  *      * -o --output: output file (default: STDOUT)
  *      * -of --output-format: (default: PARTIQL) [ION_TEXT, ION_BINARY, PARTIQL, PARTIQL_PRETTY]
+ *      * -m --monochrome: removes syntax highlighting for the REPL
```

Comment:
This option is listed under the "Non interactive only" heading. I think it should be moved up above that heading